### DISCUSSION TITLE
Use original u8g fonts in u8g2?

### DISCUSSION THREAD

**Discussion by electrichotmatt** [olikraus/u8g2#2156] [Q&A]
I'm looking to migrate from u8g to u8g2, but noticed that the fonts are different.  Is there any way to port the original fonts from u8g to u8g2? I noticed that the u8g2_font_tinyunicode_tr one is very similar to the original one I used, but has different spacing for a few characters and it throws off my old code.

Here's the font I'd like to use
https://github.com/olikraus/u8glib/blob/master/fntsrc/u8g_font_04b_03.c

Thanks!

**Comment by olikraus** [comment]
Most fonts from u8g are also there in u8g2.  In fact there is a translation table, so that you can keep the u8g name.
If your u8g font name is listed here, then it should be there in u8g2:

https://github.com/olikraus/u8g2/blob/fbeba5d1c384872caef828f762af51a28fbfc6f8/csrc/u8g2.h#L3752-L3946

**Comment by olikraus** [comment]
ok, i think you are right, I didn't port the 04 fonts: https://github.com/olikraus/u8glib/tree/master/tools/font/ttf

You need the [04B_03__.TTF](https://github.com/olikraus/u8glib/blob/master/tools/font/ttf/04B_03__.TTF) font, right?

Let me create an issue for this...

**Reply by olikraus** [reply]
https://github.com/olikraus/u8g2/issues/2157

**Reply by electrichotmatt** [reply]
Yes sir! u8g_font_04b_03r
